zerver/tests/test_email_mirror.py

```diff
@@ -232,6 +232,33 @@ def test_receive_stream_email_messages_success(self) -> None:
         self.assertEqual(get_display_recipient(message.recipient), stream.name)
         self.assertEqual(message.topic_name(), incoming_valid_message['Subject'])
 
+    def test_receive_stream_email_messages_other_header_success(self) -> None:
```

Comment:
Maybe add a comment here adding a bit of detail on what's special about this test vs. adjacent ones?  (I think it's use of `Envelope-To`?)